swarm/runtime/_claude_sdk/schemas.py

```
@@ -0,0 +1,164 @@
+
+This module contains the canonical JSON schemas used for structured output
+in the Claude SDK integration. These schemas define the expected format for:
+
+- HandoffEnvelope: Structured output for step completion handoffs
+- RoutingSignal: Structured output for routing decisions between steps
+
+These schemas are used with the SDK's output_format parameter for reliable
+JSON extraction.
+"""
+
+from typing import Any, Dict
```

Ruff (0.14.11)

`typing.Dict` is deprecated, use `dict` instead

(UP035)